ebmdatalab/generate-ebmdatalab-stats.py

```diff
@@ -1,10 +1,13 @@
 import argparse
 import datetime
 import json
+from datetime import date
 import matplotlib.pyplot as plt
 import os
 import pandas as pd
 import urllib.request
```

Comment:
@agitter the linter was very insistent that we don't actually use urllib anywhere.

Reply:
Thanks for checking.  You can delete `urllib.request`.  My initial draft needed it for the short DOI functionality, but then I realized I could import `get_short_doi_url` from Manubot instead.